lib/mix/lib/mix.ex

```diff
@@ -161,7 +161,7 @@ defmodule Mix do
 
     * `MIX_ARCHIVES` - specifies the directory into which the archives should be installed
     * `MIX_DEBUG` - outputs debug information about each task before running it
-    * `MIX_ENV` - specifies which environment should be used. see Environments
+    * `MIX_ENV` - specifies which environment should be used. See [Environments](#environments)
```

Comment:
@whatyouhide You're absolutely right. Let me fix that right now. https://github.com/elixir-lang/elixir/pull/5105